pkgs/applications/misc/actiona/default.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, wrapQtAppsHook
+
+, bluez
+, libnotify
+, libXdmcp
+, libXtst
+, opencv
+, qtbase
+, qtmultimedia
+, qtscript
+, qttools
+, qtx11extras
+, qtxmlpatterns
+
+, textToSpeechSupport ? false
+, qtspeech
+}:
+
+stdenv.mkDerivation {
+  pname = "actiona";
+  version = "3.10.1-unstable-2022-05-28";
```

```suggestion
  version = "unstable-2022-05-28";
```
See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming